gym/wrappers/pixel_observation.py

```diff
@@ -1,19 +1,52 @@
+"""Wrapper for augmenting observations by pixel values."""
+from __future__ import annotations
+
 import collections
 import copy
 from collections.abc import MutableMapping
+from typing import Any, Optional
 
 import numpy as np
 
+import gym
 from gym import ObservationWrapper, spaces
 
 STATE_KEY = "state"
 
 
 class PixelObservationWrapper(ObservationWrapper):
-    """Augment observations by pixel values."""
+    """Augment observations by pixel values.
+
+    Example:
+        >>> import gym
+        >>> env = gym.wrappers.PixelObservationWrapper(gym.make('CarRacing-v1'))
+        >>> obs = env.reset()
+        >>> obs.keys()
+        odict_keys(['pixels'])
+        >>> obs['pixels'].shape
+        (400, 600, 3)
+        >>> env = gym.wrappers.PixelObservationWrapper(gym.make('CarRacing-v1'), pixels_only=False)
+        >>> obs = env.reset()
+        >>> obs.keys()
+        odict_keys(['state', 'pixels'])
+        >>> obs['state'].shape
+        (96, 96, 3)
+        >>> obs['pixels'].shape
+        (400, 600, 3)
+        >>> env = gym.wrappers.PixelObservationWrapper(gym.make('CarRacing-v1'), pixel_keys=('obs',))
+        >>> obs = env.reset()
+        >>> obs.keys()
+        odict_keys(['obs'])
+        >>> obs['obs'].shape
```

Comment:
Having `gym.wrappers` everywhere is unnecessary noise imo

```suggestion
        >>> env = PixelObservationWrapper(gym.make('CarRacing-v1'))
        >>> obs = env.reset()
        >>> obs.keys()
        odict_keys(['pixels'])
        >>> obs['pixels'].shape
        (400, 600, 3)
        >>> env = PixelObservationWrapper(gym.make('CarRacing-v1'), pixels_only=False)
        >>> obs = env.reset()
        >>> obs.keys()
        odict_keys(['state', 'pixels'])
        >>> obs['state'].shape
        (96, 96, 3)
        >>> obs['pixels'].shape
        (400, 600, 3)
        >>> env =PixelObservationWrapper(gym.make('CarRacing-v1'), pixel_keys=('obs',))
        >>> obs = env.reset()
        >>> obs.keys()
        odict_keys(['obs'])
        >>> obs['obs'].shape
```

Reply:
Done